vdirsyncer/cli/discover.py

```diff
@@ -80,6 +80,21 @@ def collections_for_pair(status_path, pair, from_cache=True,
         get_b_discovered=b_discovered.get_self,
         _handle_collection_not_found=handle_collection_not_found
     ))
+    only_in_b = set(b_discovered.get_self().keys()) - set(a_discovered.get_self().keys())
```

Comment:
If I understand correctly, this block will delete from `b` any collections absent from `a`, right?

That makes sense if settings include:

```dosini
collections = ["from a"]
```

However, if settings are as follows, then you'd be deleting from the wrong side, right?


```dosini
collections = ["from b"]
```

Reply:
That is correct.  I uploaded new code that considers "from a" and "from b".  I do not understand all the advanced ways to set collections.